phpcs.xml.dist

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="SilverStripe">
+    <description>CodeSniffer ruleset for SilverStripe coding conventions.</description>
+
+    <rule ref="PSR2" >
+        <!-- Current exclusions -->
+        <exclude name="PSR1.Methods.CamelCapsMethodName" />
+    </rule>
+</ruleset>
```

Comment:
Since we're adding our own phpcs definitions to this module we should use it in the Travis configuration - can you remove the --standard argument from this line in .travis.yml please?

```diff
-   - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs --standard=vendor/silverstripe/framework/phpcs.xml.dist code/ tests/ ; fi
+   - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs code/ tests/ ; fi

```